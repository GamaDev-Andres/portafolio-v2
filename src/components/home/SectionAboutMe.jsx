const SectionAboutMe = () => {
  return <div>SectionAboutMe</div>
}

export default SectionAboutMe
