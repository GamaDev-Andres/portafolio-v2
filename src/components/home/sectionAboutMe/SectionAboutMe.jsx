import TitleSection from '../../TitleSection'
import MiDescripcion from './MiDescripcion'

const SectionAboutMe = () => {
  return (
    <section>
      <TitleSection textInSpan={'mi'} unOrder={true}>
        sobre
      </TitleSection>
      <MiDescripcion />
    </section>
  )
}

export default SectionAboutMe
