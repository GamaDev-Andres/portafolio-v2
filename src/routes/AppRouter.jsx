import { useState } from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import Home from '../components/home/Home';
import Project from '../components/project/Project';

function AppRouter() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/project/:id" element={<Project />} />
      </Routes>
    </BrowserRouter>
  );
}

export default AppRouter;
