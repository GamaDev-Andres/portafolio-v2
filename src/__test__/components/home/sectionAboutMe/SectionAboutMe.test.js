/**
 * @jest-environment jsdom
 */
import Enzyme from 'enzyme'
import { HelmetProvider } from 'react-helmet-async'
import { MemoryRouter } from 'react-router-dom'
import SectionAboutMe from '../../../../components/home/sectionAboutMe/SectionAboutMe'
import AppRouter from '../../../../routes/AppRouter'

describe('probando <SectionAboutMe/>', () => {
  const wrapper = Enzyme.mount(
    <SectionAboutMe />
  )
  test('debe montarse correctamente', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('TitleSection').length).toBe(1)
    expect(wrapper.find('MiDescripcion').length).toBe(1)
  })
  test('No debe montarse en ruta "project"', () => {
    const wrapper = Enzyme.mount(
      <HelmetProvider>
        <MemoryRouter initialEntries={['/project/BARBERAPP']}>
          <AppRouter />
        </MemoryRouter>
      </HelmetProvider>
    )
    expect(wrapper.find('SectionAboutMe').exists()).toBe(false)
  })
})
