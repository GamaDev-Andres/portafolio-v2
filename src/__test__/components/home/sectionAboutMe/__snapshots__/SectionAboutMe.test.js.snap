// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`probando <SectionAboutMe/> debe montarse correctamente 1`] = `
<section
  className="sm:scroll-m-24 md:scroll-m-14"
  id="sobremi"
>
  <h2
    className="my-4 uppercase font-bold text-3xl md:text-center"
  >
    sobre
     
    <span
      className="text-blue-400"
    >
      mi
    </span>
  </h2>
  <div
    className="text-fondos flex flex-col gap-4"
  >
    <article
      className="flex flex-col gap-2"
    >
      <p>
        <span
          className="text-blue-400 font-medium"
        >
          Quien soy: 
        </span>
        Mi nombre es Andres Gama, me apasiona encontrar grandes retos que me
          permitan crecer profesional y personalmente, la programación me
          permite enfocarme en esto.Actualmente tengo 22 años y sé que lograre
          que mis conocimientos sean participes de grandes, útiles y reconocidos
          proyectos.
      </p>
      <p>
        En mí podras ver a un chico ansioso por aprender y dispuesto a
          afrontar complejas curvas de aprendizaje y gracias a mi constancia y
          disciplina veras a alguien creciendo constantemente.
      </p>
    </article>
    <button
      className=" mx-auto block text-blue-400"
      onClick={[Function]}
    >
      Ver mas
       
      <i
        className="fa-solid fa-arrow-down"
      />
    </button>
  </div>
</section>
`;
